python/tank/authentication/interactive_authentication.py

```diff
@@ -141,7 +141,11 @@ def _renew_session_internal(user, credentials_handler):
             # We're the first thread, so authenticate.
             try:
                 if user.get_session_metadata() is not None:
-                    logger.debug("Attempting to renew our SSO session.")
+                    from .user import ShotgunSamlUser
```

Comment:
I had to late import the ShotgunSamlUser here, importing at the top of the file would cause a cyclical dependency